doc/api/http2.md

```diff
@@ -524,6 +524,33 @@ Used to set a callback function that is called when there is no activity on
 the `Http2Session` after `msecs` milliseconds. The given `callback` is
 registered as a listener on the `'timeout'` event.
 
+#### http2session.setConnectionWindowSize(windowSize)
+<!-- YAML
+added: v12.0.0
+-->
+
+* `windowSize` {number}
+* Returns: 0 
+In case of allocation error, a new `ERR_OUT_OF_RANGE`
+error will be thrown.
+
+Used to set the local window size (local endpoints's window size).
+The window_size is an absolute value of window size to set, rather
+than the delta.
+
+```js
+clientSession.on('connect', (session) => sendSettings(session, (s) => cb(s)));
```

Comment:
Perhaps `(s) => cb(s)` would be more clear, as documentation, if it were written with meaningful variable names (`settings`?). No need to squeeze it on 80ch; just go multi-line.